seqr/views/apis/project_categories_api_tests.py

```diff
@@ -4,15 +4,21 @@
 from seqr.views.apis.project_categories_api import update_project_categories_handler
 from seqr.views.utils.test_utils import _check_login
 
+PROJECT_GUID = 'R0001_1kg'
+
 
 class ProjectCategoriesAPITest(TestCase):
     fixtures = ['users', '1kg_project']
 
     def test_project_categories_api(self):
-        url = reverse(update_project_categories_handler, args=['R0001_1kg'])
+        url = reverse(update_project_categories_handler, args=[PROJECT_GUID])
         _check_login(self, url)
 
         response = self.client.post(url, content_type='application/json', data=json.dumps({
-            'categories': []
+            'categories': ['PC000002_categry_with_unicde', 'PC000003_categry_with_unicde']
```

Comment:
There is a typo in the second guid, so thats not an existing category, it is creating a new category with that name. Thats actually an important code path to test, but it would probably be better if you used something like `new_category` as the name instead of something that looks like it is an existing guid

Reply:
Good idea. The intention is to create new category indeed. I will update the name.